specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-01-01-preview/examples/watchlists/CreateWatchlistItem.json

```diff
@@ -4,7 +4,8 @@
     "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
     "resourceGroupName": "myRg",
     "workspaceName": "myWorkspace",
-    "watchlistAlias": "highValueAsset",
+    "operationalInsightsResourceProvider": "Microsoft.OperationalInsights",
```

Comment:
Just out of curious, I wonder why do we need to provide this operationalInsightsResourceProvider parameter value in the example? I don't see the definition in the Watchlists.json